### PR TITLE
feat: offer lower speed in MuxPlayer

### PR DESCRIPTION
**Pull request by weronika-szalas**
## Description

Added 0.5x and 0.7x speed to MuxPlayer (ticket mentioned only 0.5x but it is VERY slow)

## Issue(s)

Fixes [PUPIL-852](https://www.notion.so/oaknationalacademy/Offer-MUX-video-speed-1x-579808e7d7dc4432ba5cf8ec77f27dc0)

## How to test

1. Go to any page in the app with video in it
2. Video player should now offer 0.5x and 0.7x playback speeds

## Screenshots

How it should now look:
Screenshot 2024-11-07 at 17 42 07

### CODE REVIEW

**Review by RBrownsett — APPROVED** [review]
Approved - tested with no issues. Blog and teaching videos can have less than <1.x speed

**Review by benprotheroe — APPROVED** [review]

**Comment by weronika-szalas** [comment]
Waiting for the green light from Alex Francis/Toby to merge this

**Review by kimon-satan — APPROVED** [review]

**Pull request merged**
